tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java

```diff
@@ -22,10 +23,7 @@
 import java.util.Set;
 import java.util.TreeSet;
 
-import javax.xml.bind.DatatypeConverter;
-
 import org.unicode.cldr.util.CLDRConfig;
-import org.unicode.cldr.util.Level;
```

Comment:
any reason for removing this import?

Reply:
Originally there were multiple classes named `Level`, this disambiguates.

Reply:
VoteResolver.level vs cldr.util.Level

Reply:
@hagbard reinstated Level, PTAL